taskcluster/kinds/db/kind.yml

```diff
@@ -26,6 +27,9 @@ tasks:
         cd db && yarn test
   upgrade:
     description: yarn db:upgrade tests
+    worker:
+      env:
+        COREPACK_ENABLE_DOWNLOAD_PROMPT: "0"
```

Comment:
i'm curious what the COREPACK_ENABLE_DOWNLOAD_PROMPT does?

Reply:
Messed around with some env var settings to have `corepack` not ask me if I want to enable it and install anything on the system. I think I've worked around having to use this anyway...